web/src/pages/download/vast-linux-static-latest.deb.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+export default function Home(): JSX.Element {
+  const url = 'https://github.com/tenzir/vast/releases/latest/download/vast-linux-static.tar.gz';
+  React.useEffect(() => {
+    window.location.replace(url)
+  }, [])
+  return <></>;
+}
```

Comment:
@dit7ya if you can whip up that React component with a single breath, that'd be helpful.

Reply:
Unfortunately it is still boilerplaty - so not super sure if its worth doing. What do you think?

```diff
new file   web/src/components/ExternalLink/index.tsx
@@ -0,0 +1,9 @@
+import React from "react";
+
+const ExternalLink = ({ url }) => {
+  React.useEffect(() => {
+    window.location.replace(url);
+  }, []);
+  return <></>;
+};
+export default ExternalLink;
modified   web/src/pages/download/vast.zip.tsx
@@ -1,9 +1,12 @@
-import React from 'react';
+import ExternalLink from "@site/src/components/ExternalLink";
+import React from "react";
 
-export default function Home(): JSX.Element {
-  const url = 'https://github.com/tenzir/vast/archive/refs/heads/stable.zip';
-  React.useEffect(() => {
-    window.location.replace(url)
-  }, [])
-  return <></>;
-}
+const Element = () => {
+  return (
+    <ExternalLink
+      url={"https://github.com/tenzir/vast/archive/refs/heads/stable.zip"}
+    />
+  );
+};
+
+export default Element;
```

Reply:
Thanks, I still find it cleaner and more intuitive to read, despite the boilerplate (which fortunately remains static.) That said, does this work for you? It's not working for me.

```diff
diff --git a/web/src/pages/download/sbom.spdx.tsx b/web/src/pages/download/sbom.spdx.tsx
index c41f90da8..58e737f55 100644
--- a/web/src/pages/download/sbom.spdx.tsx
+++ b/web/src/pages/download/sbom.spdx.tsx
@@ -1,9 +1,12 @@
-import React from 'react';
+import ExternalLink from "@site/src/components/ExternalLink";
+import React from "react";
 
-export default function Home(): JSX.Element {
-  const url = 'https://github.com/tenzir/vast/releases/latest/download/VAST.spdx';
-  React.useEffect(() => {
-    window.location.replace(url)
-  }, [])
-  return <></>;
-}
+const Element = () => {
+  return (
+    <ExternalLink
+      url={"https://github.com/tenzir/vast/releases/latest/download/VAST.spdx"}
+    />
+  );
+};
+
+export default Element;
```

Reply:
Alright, I think this will work.

```jsx
import React from "react";

const ExternalLink = ({ url }) => {
  React.useEffect(() => {
    window.location.replace(url);
  }, []);
  return null;
};

export default ExternalLink;
```

Reply:
That works, thanks!